ext/standard/tests/streams/gh20286.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+GH-20286 use after destroy on userland stream_close
+--CREDITS--
+vi3tL0u1s
+--SKIPIF--
+<?php
+if (PHP_DEBUG) die('skip requires release build');
+?>
+--FILE--
+<?php
+class lib {
+  function stream_open() {
+    return true;
+  }
+
+  function stream_read($count) {
+    function a() {}
+    include('lib://');
+  }
+
+  function stream_close() {
+    include('lib://');
+  }
+}
+stream_wrapper_register('lib', lib::class);
+include('lib://test.php');
+?>
+--EXPECTF--
+
+Deprecated: Creation of dynamic property lib::$context is deprecated in %s on line %d
```

Comment:
Maybe create the `$context` prop and create empty stubs for these functions to make the warnings go away.